synIndex.R

```diff
@@ -25,44 +34,119 @@ source('params.R')
 # Get bucket params and file list
 #############
 ## Get a list of all Objects in the PRE_ETL S3 bucket 
-s3lsBucketObjects(source_bucket = paste0('s3://', PRE_ETL_BUCKET,'/'),
-                  output_file = FILE_LIST_OUTPUT)
+# s3lsBucketObjects(source_bucket = paste0('s3://', PRE_ETL_BUCKET,'/'),
+#                   output_file = FILE_LIST_OUTPUT)
 
 # Get bucket params
 bucket_params <- list(uploadType='S3',
-                    concreteType='org.sagebionetworks.repo.model.project.ExternalS3StorageLocationSetting',
-                    bucket=SOURCE_BUCKET)
+                      concreteType='org.sagebionetworks.repo.model.project.ExternalS3StorageLocationSetting',
+                      bucket=PRE_ETL_BUCKET)
 
 # The above list is just to verify/reference, 
 # since we will replicate the structure locally
 # we will work with that
 
```

Comment:
Do you use this reference for validation?  If so you could use check that you have the same content and throw an error if not.